projects/igniteui-angular/src/lib/core/styles/components/ripple/_ripple-theme.scss

```diff
@@ -56,6 +56,9 @@
 
     %igx-ripple-display {
         display: $ripple-display;
+        @if map-get($theme, variant) == 'bootstrap' or map-get($theme, variant) == 'fluent'  {
```

Comment:
Could you just check if the current theme is 'material' instead of checking what it's not to hide the ripple?